Python/other/Rearrange_the_neagtive_positive_element.py

```diff
@@ -0,0 +1,54 @@
+"""
+Given an array of positive and negative numbers, arrange them in an alternate 
+fashion such that every positive number is followed by negative and vice-versa 
+maintaining the order of appearance. 
+Number of positive and negative numbers need not be equal. 
+If there are more positive numbers they appear at the end of the array.
+If there are more negative numbers, they too appear in the end of the array.
+
+Examples : 
+
+Input:  arr[] = {1, 2, 3, -4, -1, 4}
+Output: arr[] = {-4, 1, -1, 2, 3, 4}
+
+Input:  arr[] = {-5, -2, 5, 2, 4, 7, 1, 8, 0, -8}
+output: arr[] = {-5, 5, -2, 2, -8, 4, 7, 1, 8, 0} 
```

Comment:
Remove this part as already you wrote output

Reply:
Done 👍🏻 @aakriti1318